### DISCUSSION TITLE
Change the requirements needed to reach quorum in Oracle Votings

### DISCUSSION THREAD

**Discussion by Zen-44** [idena-network/idena-docs#93] [IIPs]
The quorum represents the minimum number of votes in an Oracle Voting required to move to the public voting phase. The quorum should only be reached only when the total number of non-discriminated votes reaches the quorum value.

The proposal aims to limit the malleability of the quorum parameter by not taking into consideration votes that are not going to be counted when checking if a vote reached the quorum or not.

The full proposal can be currently read at: https://github.com/Zen-44/idena-docs/blob/IIP-6/docs/iip/iip-6.md

**Comment by Ed-Relight** [comment]
The fact that discriminated votes are even counted in a quorum sounds like a bug. Definitely needs to be fixed
